src/main/java/link/biosmarcel/baka/view/PaymentsView.java

```diff
@@ -182,44 +184,63 @@ private void classifyAllUnclassified() {
 
     private void importHandler(final Account account, final BiFunction<Account, File, List<Payment>> importer) {
         final FileChooser fileChooser = new FileChooser();
+        fileChooser.setTitle("Import for account '%s'".formatted(account.name));
+
         final var initialDir = state.data.convenienceState.importDirectories.get(account);
         if (initialDir != null) {
             fileChooser.setInitialDirectory(new File(initialDir));
         } else if (state.data.convenienceState.lastImportPath != null) {
             fileChooser.setInitialDirectory(new File(state.data.convenienceState.lastImportPath));
         }
-        final var file = fileChooser.showOpenDialog(getTabPane().getScene().getWindow());
```

Comment:
getScene will be `null`, when the Node ,TabPane in this case, is not yet part of the scene graph. that can happen in tests or during constructors usually. is this a risk in this case?

Reply:
No risk here